src/libraries/Microsoft.NETCore.Platforms/src/Microsoft.NETCore.Platforms.csproj

```diff
@@ -1,6 +1,7 @@
 <Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
     <TargetFrameworks>$(NetCoreAppToolCurrent);$(NetFrameworkToolCurrent)</TargetFrameworks>
+    <UseTargetFrameworkSDK>false</UseTargetFrameworkSDK>
```

Comment:
Since the package was rename to not have an "SDK" suffix, I would rename this to something like, `UseTargetFrameworkBuildTasks`?

Reply:
Someone hacked my personal information my whole personal data my old Gmail account my old Samsung account my all Google Drive data on my Google photos Michael all other information like Facebook messenger telegram Instagram to talk to either Snapchat WhatsApp someone hacked what's up big dog is very important because there's access not have and Instagram creator into much pain dirty up and development my Facebook on Apple developing and that my iPhone and my mama Gmail account my brother Gmail account to my house IP address and my Internet IP address my devices IP address and create you are a numberMy name on the two much create on the domain name and use someone of my view my YouTube channel something my TV hacker anything to hurt you can help me please my iCloud iPhone or your iPad or laptop anything ahead you can help me plea